components/DashboardLayout/Sidebar.tsx

```diff
@@ -53,13 +53,20 @@ const SidebarsByRole: Record<UserRoleType, React.ReactNode> = {
   Mentor: (
     <>
       <SidebarLink href="/mentor/players">Players</SidebarLink>
+      <SidebarLink href="/admin/users">Members</SidebarLink>
       {/* <SidebarLink href="/mentor/calendar">Calendar</SidebarLink> */}
       {/* <SidebarLink href="/mentor/notes">Notes</SidebarLink> */}
     </>
   ),
-  Parent: <></>,
+  Parent: (
+    <>
+      <SidebarLink href="/mentor/players">Players</SidebarLink>
+      <SidebarLink href="/admin/users">Members</SidebarLink>
+    </>
+  ),
   Player: (
     <>
+      <SidebarLink href="/donor/players">Players</SidebarLink>
       <SidebarLink href="/player/profile">Profile</SidebarLink>
```

Comment:
For Profile we want the player to be redirected to their own player profile, so that path will have to take direct to their own player profile (with id).